src/js/controller.js

```diff
@@ -769,4 +785,12 @@ export const MediaUIRequestHandlers = {
 
     media.currentTime = seekable.end(seekable.length - 1);
   },
+  MEDIA_RENDITION_REQUEST: (media, event) => {
+    if (!media?.videoRenditions) return;
```

Comment:
Let's add a console warning here. i.e. "Rendition selection not supported by this media". Silent failing here could lead to some frustration.